examples/plot_simulate_lfp.py

```diff
@@ -0,0 +1,58 @@
+"""
+=====================
+08. Plot Simulate LFP
+=====================
+
+This example demonstrates how to record local field potentials (LFPs).
+"""
+
+# Author: Nick Tolley <nicholas_tolley@brown.edu>
```

Comment:
you can use #sphinx-gallery-thumbnail to point to the second or third image as the thumbnail (see the other examples). I think either of those are more meaningful